app/templates/admin/sales/locations.hbs

```diff
@@ -26,11 +26,11 @@
             <tr>
               <td>{{entry.locationName}}</td>
               <td class="right aligned">{{entry.sales.completed.ticket_count}}</td>
-              <td class="right aligned">{{currency-symbol entry.paymentCurrency}} {{number-format entry.sales.completed.sales_total}}</td>
+              <td class="right aligned">US$ {{number-format entry.sales.completed.sales_total}}</td>
```

Comment:
This is not the valid fix @uds5501 as the line {{currency-symbol entry.paymentCurrency}} was written to correctly render the currency parser and by setting it to default 'US $' , You are not making valid fix.

Reply:
@kushthedude It actually is a valid fix . Did you read the pertaining argument in the issue itself?
The location part of the admin/sales  is not yet ready for event based currency parsing.

At a location `ABC` suppose there are 3 events. 2 of them use `USD` and one of them uses `CAD`. What would you show then, `USD`\`CAD`?  :)

Reply:
Thats what i was saying , How did you came to know location part cant render event bases currency parsing?

Reply:
How can i make them ready for currency parsing? 
@uds5501

Reply:
@kushthedude Regarding that, I have proposed #2253 to be worked during GSoC period. I will standardise the amount of other currency into USD and that should do it!